src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/login.scss

```diff
@@ -1,31 +1,181 @@
+        z-index: 30;
+        border-radius: $borderRadius;
+        @include standard-box-shadow;
+        overflow: hidden;
+        transition: height 200ms;
+
+        &.larger {
+          height: $boxHeight2;
+        }
+
+        .husky-validate-error {
+            .error-msg {
+                height: 50px;
+            }
+            &.login .bottom-container span:not(.website-switch) {
+              color: $black;
+            }
+        }
+
+        .frames {
+            height: 100%;
+            width: 50px;
+            white-space: nowrap;
+            position: relative;
+            top: 0px;
+            left: 0px;
+            transition: left 300ms;
+        }
+
+        .frame {
+            height: 100%;
+            width: $boxWidth;
+            display: inline-block;
+            vertical-align: top;
+            white-space: normal;
+            position: relative;
+            padding: 0 50px;
+        }
+
+        .inputs {
+            .form-element {
+                padding-left: 5px;
+
+            }
+        }
+
+        .error-msg {
+            color: $red;
+            display: block;
+            height: 0px;
+            overflow: hidden;
+            transition: height 200ms;
+        }
+
+        .login-logo {
+            width: 100%;
+            height: 180px;
+            background: url("../img/logo.png") no-repeat center center;
+            background-size: auto 122px;
+            margin-bottom: 10px;
+        }
+        .login-loader {
+            float: right;
+            margin: 20px 15px 0 0;
+        }
+        .reset-mail-msg .login-loader {
+            float: none;
+            margin-right: 0;
+            margin-left: 0;
+        }
+        .btn.action {
+            float: right;
+            margin-top: 10px;
+            padding-left: 45px;
+            padding-right: 45px;
+        }
+
+        .bottom-container {
+            position: absolute;
+            bottom: 0px;
+            left: 0px;
+            width: 100%;
+            padding: 0 50px;
+            @include clear-float;
+
+            span {
+                display: block;
+                padding: 10px 0 10px 10px;
+                cursor: pointer;
+                color: $grayLight;
+                float: right;
+                &:hover {
+                  color: $black;
+                }
+            }
+            .website-switch {
+                float: left;
+                padding-left: 0;
+            }
+        }
+
+        .reset-mail {
+            .bottom-container span:not(.website-switch) {
+                color: $black;
+            }
+        }
+
+        .reset-mail-msg {
+            text-align: center;
+            .msg {
+                display: block;
+                color: $grayLight;
+                padding-bottom: 20px;
+            }
+            .btn.action {
+              float: none;
+              margin: 0;
+              padding: 0 15px;
+            }
+        }
```

Comment:
yes. at least in husky there is an own file "btn.scss" with a .btn class which is use all over the place